### PR TITLE
Add the -Q option to ag

### PR DESCRIPTION
**Pull request by Florent2**

### CODE REVIEW

**Comment by igorshubovych** [comment]
Is it that important use case? Literal expression is just subtype of regular expression.

**Comment by felixonmars** [comment]
Good to know, but I didn't ever use that myself :joy_cat: 

Actually I want to mention the smart-case somewhere on this page, but didn't find a good way to...

File: pages/common/ag.md [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 `ag '^ba(r|z)$'`
 
+- Find files whose contents match a literal expression, not a regular expression:
+
+`ag -Q '.bar`
```
Comment (ostera) [inline pages/common/ag.md]: missing closing single quote

Comment (Florent2) [inline pages/common/ag.md]: fixed

**Comment by igorshubovych** [comment]
@Florent2 i see you made amended commit.

Do you actively use ag literal expressions instead of just regular expressions?

**Comment by Florent2** [comment]
@igorshubovych I do, but I understand that it may not be an important use case, so no problem if you don't merge it :)

**Comment by igorshubovych** [comment]
Good. Because we want to keep as less examples as possible having only the most essential cases. "Less is better" is the main goal of this project.

I do not think this should be merged. But let's wait a bit for others' opinions.

**Comment by felixonmars** [comment]
Thanks for the PR, but I will agree with @igorshubovych :)

**Comment by igorshubovych** [comment]
@Florent2,

Thanks for the PR. Keep up the good work.

**Pull request closed without merging**